server/src/main/kotlin/com/kamelia/hedera/rest/file/Service.kt

```diff
@@ -190,21 +190,31 @@ object FileService {
         userId: UUID,
         dto: FileUpdateDTO,
     ): ActionResponse<FileRepresentationDTO> = Connection.transaction {
-        val file = File.findById(fileId) ?: throw FileNotFoundException()
-        val user = User[userId]
+        validate {
+            val file = File.findById(fileId) ?: throw FileNotFoundException()
+            val user = User[userId]
 
-        val oldName = file.name
-        val updatedFile = updateFile(file, user, FileUpdateDTO(name = dto.name))
-        val payload = updatedFile.toRepresentationDTO()
+            if (dto.name == null) {
+                raiseError("name", Errors.Files.Name.MISSING_NAME)
+            } else {
+                if (dto.name.length > 255) raiseError("name", Errors.Files.Name.NAME_TOO_LONG)
```

Comment:
hesitant to extract this magic number

also, isn't 255 too short ? I know we HAVE to have a limit, but i figured 255 might be not enough, 511 or 1023 ?

Reply:
According to Google, Windows file **path** length is limited to 256 characters. I think 255 characters for a file name is WAY enough (try to write such a name, you will understand)

Reply:
fair enough